### PR TITLE
Fixes miscellaneous i18n problèmes

### LINKED ISSUE #4444: miscellanous french issues
<sup>ℹ️ [Figma (root)][figma] | [Sitemap][sitemap]</sup>
<sup>_This initial comment is collaborative and open to modification by all._</sup>

## Description
Various French-side typos and oopsies
May edit more in as I see them, since these are so small it is easier to try and collect a few into one spot

## Screenshot
/profile page
image

## Acceptance Criteria
- [ ] en francais 

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap

### PR DESCRIPTION
**Pull request by mnigh**
Resolves #4444.

### CODE REVIEW

**Comment by mnigh** [comment]
String sent to be translated into French:

```
{
  defaultMessage: "(Request is still pending)",
  id: "FxceQZ",
  description: "Text for when date done is pending in the manager info section of the single search request view.",
}
```

File: frontend/talentsearch/src/js/lang/fr.json [hunk 1]
```diff
@@ -795,7 +795,7 @@
     "description": "Button label for submit button on create account form."
   },
   "HZB4iF": {
-    "defaultMessage": "\"Je m'identifie comme une femme\"",
+    "defaultMessage": "« Je m'identifie comme une femme »",
     "description": "Title for when someone indicates they are a woman on their profile"
```
Comment (esizer) [inline frontend/talentsearch/src/js/lang/fr.json:799]: ⛳️ for removal.

**Review by esizer — COMMENTED** [review]
Looking good! I marked some strings I couldn't find matching IDs for. Not sure we care yet since we have an issue to re-enable it #4305 🤷 

If that is out of scope let me know and I can approve.

**Comment by mnigh** [comment]
> Looking good! I marked some strings I couldn't find matching IDs for. Not sure we care yet since we have an issue to re-enable it #4305 🤷
> 
> If that is out of scope let me know and I can approve.

Indeed, there are definitely strings that are not used but are still in our codebase. #4305 will make sure orphans like this do not remain, but #4489 is the issue to remove them. I would say its out of the scope for this issue which deals with i18n and grammar.

**Review by esizer — APPROVED** [review]
LGTM 👍

**Pull request merged**